src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,231 @@
+                                    ( css
+                                      { marginBottom: int 8
+                                      , maxWidth: str "calc(100% - 24px)"
+                                      }
+                                    )
+                                $ _ { content = [ title ]
+                                    }
+                              _ -> mempty
+                          , box
+                            $ S._align S.Center
+                            $ S._row
+                            $ style_
+                                ( css
+                                  { flex: str "1" }
+                                )
+                            $ _ { content =
+                                    [ case props.icon of
+                                        Just icon ->
+                                          box
+                                          $ S._alignSelf S.Start
+                                          $ S.onDesktop (S._alignSelf S.Center)
+                                          $ style_
+                                              ( fold
+                                                  [ css
+                                                      { marginRight: int 16
+                                                      }
+                                                  , desktopQuery $ css
+                                                      { alignSelf: str "center"
+                                                      }
+                                                  ]
+                                              )
+                                          $ _ { content = [ icon ]
+                                              }
+                                        _ -> mempty
+                                    ]
+                                    <> props.content
+                                }
+                          ]
+                      }
+                ,  Monoid.guard props.dismissable
+                    $ E.element R.div'
+                      { children:
+                          [ Icon.icon
+                              { style: R.css { fontSize: "12px" }
+                              , type_: Icon.Remove
+                              }
+                          ]
+                      , className: ""
+                      , css: theme # toCSS dismissButtonStyle
+                      , onClick: capture_ $ setVisible \_ -> false
+                      }
+                ]
+            }
+  where
+    defaults :: Record BannerProps
+    defaults =
+      { component: Banner
+      , title: Nothing
+      , content: []
+      , dismissable: false
+      , icon: Nothing
+      }
+
+    dismissButtonStyle :: forall props. PropsModifier props
+    dismissButtonStyle =
+      S._interactive
+      <<< style \(LumiTheme { colors }) ->
+            css
+              { position: str "absolute"
```

Comment:
This can lead to some nasty positioning bugs and is not necessary. Can we just use normal margins or padding instead? This might require changing how align-items work in the parent.